charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.39.1
+version: 0.40.0
```

Comment:
helm version should be 0.39.2 as there is no breaking change or version upgrade.

Reply:
Version updated.